docs/model/cgw.rst

```diff
@@ -0,0 +1,5 @@
+Push items: cgw
+================
```

Comment:
If the underline doesn't match exactly the section title in length, I think it generates a sphinx warning.

```suggestion
===============
```